static/partials/feedback.html

```diff
@@ -0,0 +1,141 @@
+<div class="container" xmlns="http://www.w3.org/1999/html">
+
+    <div class="col-md-6 col-md-offset-3">
+        <h1>{{ vm.user.name }} {{ vm.user.surname }}</h1><small>{{ vm.user.email }}</small>
+        <h3 ng-hide="vm.user.lang === 'pl'">Language:</h3>
+        <h3 ng-show="vm.user.lang === 'pl'">Język:</h3>
+        <form name="form" ng-submit="vm.update_user()" role="form">
+            <div class="form-group">
+                <label for="lang">
+                    <input class="form-inline text-left" type="radio"
+                           name="lang" id="lang" value="pl"
+                           ng-model="vm.user.lang"
+                           required/> Polski
+                    <input class="form-inline float-right" type="radio"
+                           name="lang" id="lang" value="en"
+                           ng-model="vm.user.lang"
+                           required/> English
+                </label>
+            </div>
+
+            <div class="form-group"
+                 ng-class="{ 'has-error': form.city.$dirty && form.password.$error.required }">
+                <label for="city" ng-if="vm.user.lang === 'en'">City</label>
+                <label for="city" ng-if="vm.user.lang === 'pl'">Miejscowość</label>
```

Comment:
Lepiej takie rzeczy robić tak:
<label for="city">{{ vm.user.lang === 'pl' ? 'Miejscowość' : 'City' }}</label>